commons/passive/garmin/garmin_generic_accelerometer.avsc

```diff
@@ -0,0 +1,46 @@
+{
+   "namespace":"org.radarcns.passive.garmin",
+   "type":"record",
+   "name":"GarminGenericAccelerometer",
+   "doc":"Data from accelerometer sensor.",
+   "fields":[
+      {
+
+         "name":"time",
+         "type":"double",
+         "doc":"Device timestamp in UTC (s)."
+      },
+       {
+
+               "name":"timeReceived",
+               "type":"double",
+               "doc":"Device receiver timestamp in UTC (s)."
+            },
+      {
+         "name":"timestamp13",
+         "type":"double",
+         "doc":"A timestamp13 value which is a 13 bit local time, the time when this sample was gathered from the Garmin device (useful for calculating jitter in the streaming data)."
+      },
+
+      {
+         "name":"x",
+         "type":"float",
+         "doc":"Acceleration in the x-axis (g)."
+      },
+      {
+         "name":"y",
+         "type":"float",
+         "doc":"Acceleration in the y-axis (g)."
+      },
+      {
+         "name":"z",
+         "type":"float",
+         "doc":"Acceleration in the z-axis (g)."
+      },
+      {
+         "name":"samplingRate",
+         "type" : "int",
+         "doc": "Sampling rate."
```

Comment:
In samples per second?

Reply:
That is correct. Updated the doc to reflect it.